docs/en/datasets/recog.md

```diff
@@ -79,26 +79,31 @@
 │   │   ├── annotations
 │   │   ├── train_label.txt
 │   │   ├── test_label.txt
+│   ├── coco_textv2
+│   │   ├── crops
+│   │   ├── train_label.jsonl
+│   │   ├── val_label.jsonl
```

Comment:
There is one thing that confuses me. No matter we are running recognition or detection script, files of `imgs` and `annotations` will always exist. But we don't need them in recognition. So, should we list their names here?

Reply:
> There is one thing that confuses me. No matter we are running recognition or detection script, files of `imgs` and `annotations` will always exist. But we don't need them in recognition. So, should we list their names here?

That's also my concern. I only list the related dirs for the certain task (like only cropped imgs & labels for recog, while imgs, annotation for det). What do you @gaotongxiao think?

Reply:
What @xinke-wang did is right. We don't need to list out all the files that are irrelevant to this task.